### PR TITLE
add about_page

### PR DESCRIPTION
**Pull request by gittafia**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ghost) [inline about/index.html]: Please change the title.

File: about/index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+</head>
+<body>
+    <h1>Почему я выбрала направление Front-end разработки</h1>
+    <p>Lorem <strong>ipsum</strong> dolor sit amet consectetur, adipisicing elit. <em>Commodi</em> nulla inventore similique animi perspiciatis voluptate architecto iste doloremque accusamus, illo repudiandae atque, pariatur rerum doloribus! Voluptatem optio minus itaque praesentium.</p>
```
Comment (ghost) [inline about/index.html]: If the line became very long, go to the next line.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <br>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: about/index.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    <h2>Мои сильные качества разработчика</h2>
+    <p>Lorem ipsum dolor sit amet consectetur, <mark>adipisicing</mark> elit. Aliquid, temporibus corporis qui cum optio commodi quia laboriosam praesentium recusandae, magni tempora accusamus molestiae at! <del>Sapiente</del> quasi soluta quod? Magnam, eos!</p>
+    <br>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: about/index.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    <h3>Чего я хочу добиться?</h3>
+    <p>Lorem ipsum dolor sit amet <ins>consectetur</ins> adipisicing elit. Id rerum deleniti possimus ad maiores recusandae! Sunt neque accusamus, atque dolores saepe omnis esse, officia dolorum <del>hic minima vitae beatae</del> <ins>voluptatibus?</ins></p>
+    <hr>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: about/index.html [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    <blockquote> H<sub>2</sub>O Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet animi similique culpa labore aliquid. Ipsa iure porro tempora explicabo sed, veritatis magni tempore, perspiciatis dignissimos laudantium cumque voluptatem ipsum eaque 5<sup>2</sup>.</blockquote>
+    <br>
+    <hr>
+    <a href="https://github.com/gittafia">Ссылка на мой аккаунт в GitHub &#x1F43E;</a>
+    <a href="mailto:poleyko.ta@gmail.com">Ссылка на мою почту &#x2709;</a>
+    <br>
+    <p>Всего доброго &#x270C;</p>
+
+
```
Comment (ghost) [inline about/index.html]: Delete empty lines.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Моя первая страница</h1>
+    <p><a href="about/index.html">Об авторе</a></p>
```
Comment (ghost) [inline index.html]:
```suggestion
    <p><a href="./about/index.html">Об авторе</a></p>
```

File: about/index.html [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    <br>
+    <h2>Мои сильные качества разработчика</h2>
+    <p>Lorem ipsum dolor sit amet consectetur, <mark>adipisicing</mark> elit. Aliquid, temporibus corporis qui cum optio commodi quia laboriosam praesentium recusandae, magni tempora accusamus molestiae at! <del>Sapiente</del> quasi soluta quod? Magnam, eos!</p>
+    <br>
+    <h3>Чего я хочу добиться?</h3>
+    <p>Lorem ipsum dolor sit amet <ins>consectetur</ins> adipisicing elit. Id rerum deleniti possimus ad maiores recusandae! Sunt neque accusamus, atque dolores saepe omnis esse, officia dolorum <del>hic minima vitae beatae</del> <ins>voluptatibus?</ins></p>
+    <hr>
+    <blockquote> H<sub>2</sub>O Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet animi similique culpa labore aliquid. Ipsa iure porro tempora explicabo sed, veritatis magni tempore, perspiciatis dignissimos laudantium cumque voluptatem ipsum eaque 5<sup>2</sup>.</blockquote>
+    <br>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: about/index.html [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    <hr>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ghost) [inline index.html]: Please change the title.

File: about/index.html (hunk 1 above)
Comment (gittafia) [reply about/index.html]: Changed the title.

File: about/index.html (hunk 2 above)
Comment (gittafia) [reply about/index.html]: Done.

File: about/index.html (hunk 3 above)
Comment (gittafia) [reply about/index.html]: Done.

File: about/index.html (hunk 4 above)
Comment (gittafia) [reply about/index.html]: Done.

File: about/index.html (hunk 5 above)
Comment (gittafia) [reply about/index.html]: Done.

File: about/index.html (hunk 6 above)
Comment (gittafia) [reply about/index.html]: Done.

File: index.html (hunk 7 above)
Comment (gittafia) [reply index.html]: Done.

File: about/index.html (hunk 8 above)
Comment (gittafia) [reply about/index.html]: Done.

File: about/index.html (hunk 9 above)
Comment (gittafia) [reply about/index.html]: Done.

File: about/index.html [hunk 11]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ghost) [inline about/index.html]: All meta tags should be closed.

File: index.html (hunk 10 above)
Comment (ghost) [reply index.html]: Doesn't fix.

File: index.html (hunk 7 above)
Comment (ghost) [reply index.html]: Doesn't fix.

File: about/index.html [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+    <title>Моя первая страница</title>
+</head>
+<body>
+    <h1>Почему я выбрала направление Front-end разработки</h1>
+    <p>Lorem <strong>ipsum</strong> dolor sit amet consectetur, adipisicing elit. <em>Commodi</em> nulla inventore similique animi perspiciatis voluptate architecto iste doloremque accusamus, illo repudiandae atque, pariatur rerum doloribus! Voluptatem optio minus itaque praesentium.</p>
+    </br>
```
Comment (ghost) [inline about/index.html]: Are you sure what it is right syntactic? See another tag too.

Comment (gittafia) [reply about/index.html]: Fixed.

File: about/index.html (hunk 11 above)
Comment (gittafia) [reply about/index.html]: Done.

File: index.html (hunk 10 above)
Comment (gittafia) [reply index.html]: Done.

**Comment by ghost** [comment]
Good job.

**Pull request merged**